orchestrator/src/lib.rs

```diff
@@ -194,12 +248,8 @@ where
     KEY: serde::Serialize,
     ELECTION: serde::Serialize,
 {
-    let api_toml = toml::from_str::<toml::Value>(include_str!(concat!(
```

Comment:
let's leave the instantiation of the API as it is, if we create the API from a file, external crates wont be able to run the orchestrator

Reply:
Yes! Makes sense. Updated it